features/exploredash/src/main/java/org/dash/wallet/features/exploredash/data/ctxspend/model/GiftCardResponse.kt

```diff
@@ -25,9 +25,14 @@ data class GiftCardResponse(
     @SerializedName("number") val cardNumber: String? = "",
     @SerializedName("pin") val cardPin: String? = "",
 
-    @SerializedName("cryptoAmount") val cryptoAmount: String? = "",
-    @SerializedName("cryptoCurrency") val cryptoCurrency: String? = "",
-    @SerializedName("fiatAmount") val fiatAmount: String? = "",
-    @SerializedName("fiatCurrency") val fiatCurrency: String? = "",
+    @SerializedName("paymentCryptoAmount") val cryptoAmount: String? = "",
+    @SerializedName("paymentCryptoCurrency") val cryptoCurrency: String? = "",
+    val paymentCryptoNetwork: String = "",
+    val paymentId: String = "",
+    val percentDiscount: String = "",
+    val rate: String = "",
+    val redeemUrl: String = "",
+    @SerializedName("paymentFiatAmount") val fiatAmount: String? = "",
+    @SerializedName("paymentFiatCurrency") val fiatCurrency: String? = "",
```

Comment:
CTX submitted these structures with `@SerializedName` that matched the field name.  Later new fields were added and some names were changed.